applications/obsstats_ts/update_omb_stats.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#SBATCH --account=da-cpu
+#SBATCH --qos=debug
+#SBATCH --output=plot.out
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --time=00:05:00
+
+source /scratch3/NCEPDEV/da/Guillaume.Vernieres/venvs/gdas-marine-viz/bin/activate
+
+# Link experiments
+# Link experiments (skip if the links already exist)
+for src in \
+    "/scratch4/NCEPDEV/global/John.Steffen/hpss_arch/cp4.02d-parallel-obsforge" \
```

Comment:
This expt is no longer running and it can be removed or replaced. The new parallel, 3dvar expt is named cp4.03-parallel-3dvar